app/Services/Servers/SuspensionService.php

```diff
@@ -55,11 +56,7 @@ public function toggle(Server $server, string $action = self::ACTION_SUSPEND): v
         try {
             // Tell daemon to re-sync the server state.
             $this->daemonServerRepository->setServer($server)->sync();
-        } catch (\Exception $exception) {
-            // Rollback the server's suspension status if daemon fails to sync the server.
-            $server->update([
-                'status' => $isSuspending ? null : ServerState::Suspended,
-            ]);
+        } catch (ConnectionException $exception) {
```

Comment:
I agree with this change. Yes, wings does need to know that the server is suspended, but if it's currently not connected, it will simply sync that correctly when it reconnects!